tools/ci_download_cockroachdb

```diff
@@ -101,12 +96,17 @@ function fail
 function configure_os
 {
 	echo "current directory: $PWD"
-	echo "configuring based on OS: \"$1\""
+	echo "configuring based on OS: \"$1\" \"$2\""
 	case "$1" in
 		darwin*)
-			CIDL_BUILD="darwin-10.9-amd64"
+			if [[ "$2" = "aarch64" ]]; then
```

Comment:
Technically should be `==`, although `=` works.
On my mac, `$HOSTTYPE` is arm64, not aarch64.
```
pptang:omicron:main% bash

bash-3.2$ echo $OSTYPE
darwin23
bash-3.2$ echo $HOSTTYPE
arm64
bash-3.2$
```

Reply:
hmmm

```
$ bash
bash-5.2$ echo $HOSTTYPE
aarch64
bash-5.2$ echo $OSTYPE
darwin23.2.0
```

Reply:
That should work. Looks like we could also use `uname -m`, which returns `arm64` consistently, but that is more complicated than sticking to `$HOSTTYPE`.

Reply:
Apple's refusal to ship a GPLv3-licensed Bash strikes again!!